portfolio/src/main/webapp/resources.html

```diff
@@ -0,0 +1,188 @@
+    <meta http-equiv="Refresh" content=" url='https://actionnetwork.org/fundraising/louisville-community-bail-fund/'" />
+  <p><a href="https://actionnetwork.org/fundraising/louisville-community-bail-fund/">Bail Fund</a> for people protesting in Breonna Taylor's community.</p>
+
+  <p style="color:#283cc8; font-size:30px"> Justice for Elijah McClain</p>
+  <meta http-equiv="Refresh" content=" url='https://www.gofundme.com/f/elijah-mcclain?utm_source=customer&utm_campaign=p_cp+share-sheet&utm_medium=copy_link-tip'" />
+  <p><a href="https://www.gofundme.com/f/elijah-mcclain?utm_source=customer&utm_campaign=p_cp+share-sheet&utm_medium=copy_link-tip">GoFundMe</a> For Elijah McClain's mother.</p>
+
+  <p style="color:#283cc8; font-size:30px"> Justice for Ahmaud Arbery</p>
+  <meta http-equiv="Refresh" content=" url='https://www.gofundme.com/f/i-run-with-maud'" />
+  <p><a href="https://www.gofundme.com/f/i-run-with-maud">GoFundMe</a> For Ahmaud Arbery's mother and family.</p>
+   
+
+</body>
+</div>
+
+<div id="Actionables" class="tabcontent">
+<body>
+  <h4 style="color:#283cc8; font-size:30px">Actionables</h4>
+  <p style="color:#283cc8; font-size:30px" >Actionable resources</p>
+
+  <img src = "/images/FullSizeRender.jpg">
+
+  <p style="color:#283cc8; font-size:30px"> Websites with actionables:</p>
+  <meta http-equiv="Refresh" content=" url='https://www.eventbrite.com/d/ny--new-york/rallies/'" />
+  <p style="color:#283cc8; font-size:30px"><a href="https://www.eventbrite.com/d/ny--new-york/rallies/">Find a rally</a> near me.</p>
+
+
+  <meta http-equiv="Refresh" content=" url='https://www.amnesty.org/en/get-involved/take-action/'" />
+  <p style="color:#283cc8; font-size:30px"><a href="https://www.amnesty.org/en/get-involved/take-action/">Redirect me</a> to amnesty.org. Contains comprehensive list of actionables.</p>
+  
+
+  <meta http-equiv="Refresh" content=" url='https://www.forbreonna.com/'" />
+  <p style="color:#283cc8; font-size:30px"><a href="https://www.amnesty.org/en/get-involved/take-action/">Call and email</a> authorities in Breonna Taylor's community.</p>
+  
+
+</body>
+</div>
+
+<div id="Education" class="tabcontent">
+<body>
+  <h5 style="color:#283cc8; font-size:30px">Education</h5>
+  <p style="color:#283cc8; font-size:30px">Education resources</p>
+
+  <meta http-equiv="Refresh" content=" url='https://drive.google.com/drive/u/0/folders/1LGslwJwhXvpVnDgw0uC-n794l6EGzpuH'" />
+  <p style="color:#283cc8; font-size:30px"><a href="https://drive.google.com/drive/u/0/folders/1LGslwJwhXvpVnDgw0uC-n794l6EGzpuH">Read</a> this comprehensive BLM curriculum resource guide.</p>
+  <p> Contains: Postsecondary/ Adult education, multilingual resources, early childhood, elementary school, middle school, high school, and general resources.
+
+
+</body>
+</div>
+
+<script>
```

Comment:
Would it be pretty straightforward to move this into a separate file? I prefer to keep javascript and html separate.

I think you can do it by moving this into a file called "resources.js" and adding `<script src="resources.js"></script>` near the top of this html file, like what's at the top of the map.html file: https://github.com/jamilao/sps-team-project/blob/eb0b5340fa7b311bf1d5b0eda15617b90e063bcb/portfolio/src/main/webapp/map.html#L4